src/components/Card/Card.vue

```diff
@@ -30,16 +30,44 @@ export default {
     weatherIconSrc: String,
     id: Number
   },
+  data: () => ({
+    weatherIcons: [
+      { id: '01d', desc: 'Clear skies sunny weather icon' },
+      { id: '01n', desc: 'Clear skies moon weather icon' },
+      { id: '02d', desc: 'Few clouds sun weather icon' },
+      { id: '02n', desc: 'Few clounds moon weather icon' },
+      { id: '03d', desc: 'Scattered clouds weather icon' },
+      { id: '04d', desc: 'Broken clouds weather icon' },
+      { id: '09d', desc: 'Shower rain weather icon' },
+      { id: '10d', desc: 'Rainy day weather icon' },
+      { id: '10n', desc: 'Rainy night weather icon' },
+      { id: '11d', desc: 'Thunderstorm weather icon' },
+      { id: '13d', desc: 'Snow weather icon' },
+      { id: '50d', desc: 'Misty Weather Icon' }
+    ],
+    weatherIconAlt: ''
+  }),
```

Comment:
I love this way of populating the alt descriptions for our weather icons - really smart!
Only question is whether we would want get feedback on how helpfully descriptive the text is, and if we'd want to make any adjustments for that reason (ie. I'm not sure I'd know what to picture for 'Misty Weather Icon', but agreed that it should effectively be describing the actual weather occurring whenever this icon would be applied)